src/screens/TabScreen/index.js

```diff
@@ -13,6 +13,11 @@ const Tab = createBottomTabNavigator();
 
 export default function TabScreen(props) {
 
+	const data = [
```

Comment:
is this used anywhere? Even if used do we need to send it from this screen?

Reply:
I passed it from tab screen at first but I used it in notification only to change the colour of the notification icon so I added it directly to the notification file. I forgot to remove this one but I already remove it now. Thank you.